manifests/rhoai/shared/odhdashboardconfig/odhdashboardconfig.yaml

```diff
@@ -28,6 +28,7 @@ spec:
     disableModelMesh: false
     disableDistributedWorkloads: false
     disableModelRegistry: false
+    disableModelRegistrySecureDB: true
```

Comment:
Also following [this thread](https://redhat-internal.slack.com/archives/C05SMJ09DD2/p1730922294428399), should we modify this file?

Reply:
asking about this in the relevant thread

Reply:
We have [an answer](https://redhat-internal.slack.com/archives/C05SMJ09DD2/p1733754222655599?thread_ts=1730922294.428399&cid=C05SMJ09DD2)

Reply:
i added EA to the flag, big convo but i think that was what is expected out of it

Reply:
The [message](https://redhat-internal.slack.com/archives/C05SMJ09DD2/p1733754298033679?thread_ts=1730922294.428399&cid=C05SMJ09DD2) changed.. seems we don't need `EA` .. we don't need to add the flag to RHOAI OdhDashboardConfig during development .. we just need to add it at the end .. when the feature is going live and enable it then.

Reply:
removed EA, removed flag from odhdashboardconfig